docs/admin-guide.md

```diff
@@ -0,0 +1,92 @@
+## Admin Guide
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Webhooks](#web-hooks)
+- [Slash Commands](../README.md/#slash-commands)
+- [Onboard Users](#onboard-users)
+- [FAQ](#faq)
+- [Get Help](#get-help)
+
+## Prerequisites 
+
+This guide is intended for Mattermost System Admins setting up the CircleCI plugin and Mattermost users who want information about the plugin functionality.
+
+This guide assumes you have:
+
+* A project, hosted on github.com or bitbucket.org.
+* A CircleCI SaaS account, which has access to the projects/org you want to interact with.
+* Mattermost self-hosted: A Mattermost server running v5.12 or higher, with a configured Site URL. v5.24 or higher is recommended to have the autocomplete feature.
+
+## Installation
+
+### Marketplace installation
```

Comment:
Please remove this section on Marketplace installation since it's no longer applicable. Thanks!

Reply:
@cwarnermm I have removed Marketplace installation from the admin-guide in my latest commit.
I have a doubt, did we remove al the plugins from marketplace or only certain plugins have been removed ?

Reply:
Plugins that our community manages have been removed from the Mattermost Marketplace. Plugins that Mattermost manages will continue to be posted there.

Reply:
Thanks for confirming, @cwarnermm!